### ISSUE TITLE
How to interpret the output?

### ISSUE THREAD

**Issue by YiweiZhu** [gaochenyi/CC-PLM#1]
Hi, @gaochenyi 
    Thanks for developing this great tool! I learn it from a recently-published paper in [Nature Microbiology](https://www.nature.com/articles/s41564-022-01204-x). I think CC-DCA is very useful for studying bacterial genome epistasis, even if I do not understand its mathematic details. I just reproduce the results in your paper about 3000 _S. pneumoniae_ genomes. Finally, I get a wide table and transpose into a long one. I think the first two columns represent the loci in the MSA file, but what does the third column mean? Do these values indicate the strength of interaction？
![1696570016550](https://github.com/gaochenyi/CC-PLM/assets/41197115/9c5fb298-86c9-4905-9cf5-45eac3b644c4)

**Comment by gaochenyi** [comment]
Yes, the third column lists strength of interaction between locus i and locus j, which is calculated by a formula (see the code or our paper for details).

Note different formula can be used for scoring. And different scoring scheme leads to different scores, so basically you should not think an interaction with score 0.6 is stronger than an interaction with score 0.58. But an interaction with score 1 is highly probably stronger than an interaction with score 0.01.

Predictions by this modeling scheme can only be validated by experiments.

Hope this tool gives you good predictions
